### PR TITLE
feat(onboarding): enable another iteration of onboarding guide experiment

### PR DESCRIPTION
**Pull request by BohdanOne**
**Resolves**: [DT-2524](https://linear.app/prismic/issue/DT-2524/aapm-i-can-see-a-new-experiment-with-the-shared-onboarding-guide)

### Description

In another iteration of onboarding guide experiment we display either shared onboarding guide or nothing.
I've updated key of the experiment to match new setup in Amplitude and removed code related to old guides that are no longer needed.

### Checklist

- [ ] A comprehensive Linear ticket, providing sufficient context and details to facilitate the review of the PR, is linked to the PR.
- [ ] If my changes require tests, I added them.
- [ ] If my changes affect backward compatibility, it has been discussed.
- [ ] If my changes require an update to the CONTRIBUTING.md guide, I updated it.

### How to QA [^1]

[^1]:
	Please use these labels when submitting a review:
	:question: #ask:&ensp;Ask a question.
	:bulb: #idea:&ensp;Suggest an idea.
	:warning: #issue:&ensp;Strongly suggest a change.
	:tada: #nice:&ensp;Share a compliment.

### CODE REVIEW

File: packages/slice-machine/src/features/onboarding/OnboardingGuide.tsx [hunk 1]
```diff
@@ -1,29 +1,24 @@
...
 
 import { SharedOnboardingGuide } from "./SharedOnboardingGuide";
-import { SliceMachineOnboardingGuide } from "./SliceMachineOnboardingGuide/SliceMachineOnboardingGuide";
 import { useSharedOnboardingExperiment } from "./useSharedOnboardingExperiment";
 
 export function OnboardingGuide() {
   const isVisible = useIsOnboardingGuideVisible();
   const isSharedExperimentEligible = useSharedOnboardingExperiment().eligible;
 
   if (!isVisible) return null;
```
Comment (xrutayisire) [inline packages/slice-machine/src/features/onboarding/OnboardingGuide.tsx]: 💡 You could put all `null` cases all together:

```suggestion
  if (!isVisible || !isSharedExperimentEligible) return null;
```

WDYT?

**Review by xrutayisire — APPROVED** [review]
LGTM, just one small comment.

Love to see that many removals, huge win!

I guess this removal required some QA to confirm nothing is broken. Even if we have e2e tests.

**Comment by xrutayisire** [comment]
Btw I guess you confirm with Product the old onboarding guide should never come back.

**Comment by BohdanOne** [comment]
> Btw I guess you confirm with Product the old onboarding guide should never come back.

Yes, asked Come and he confirmed we don't want to keep any of the older versions of onboarding.
I did some QA locally, will still do more on alpha.

File: packages/slice-machine/src/features/onboarding/OnboardingGuide.tsx (hunk 1 above)
Comment (BohdanOne) [reply packages/slice-machine/src/features/onboarding/OnboardingGuide.tsx]: true, changed

**Pull request merged**